druid-handler/README.md

```diff
@@ -0,0 +1,3 @@
+# Druid Storage Handler
+
+The handler will try to create the druid segments table if it does not exist, this will be done as part of the pre-create process.
```

Comment:
@b-slim , could you add the link to the wiki rather that this text (as Druid Storage Handler does much more than just creating segments in Druid)?